lib/AST/ASTDumper.cpp

```diff
@@ -2047,6 +2047,8 @@ class PrintExpr : public ExprVisitor<PrintExpr> {
   }
   void visitArrayExpr(ArrayExpr *E) {
     printCommon(E, "array_expr");
+    PrintWithColorRAII(OS, LiteralValueColor) << " initializer=";
```

Comment:
What if there is no initializer yet? Doe the empty ConcreteDeclRef support a dump method?

Reply:
This prints: **NULL**.